### PR TITLE
@jory: Deploys

### PR DESCRIPTION
**Pull request by ashfurrow**
Lots of things here. Basically this is the first deploy of Eidolon in a few months, and I had to update a few things. I'll make notes below. 

I couldn't get Hockey to upload without adding this to the Fastfile:

```rb
require 'openssl'
OpenSSL::SSL::VERIFY_PEER = OpenSSL::SSL::VERIFY_NONE
```

Which is a bad idea, so I didn't commit them. It's likely something wrong with my computer, so I can address that separately.

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -39,9 +39,7 @@ pod 'ISO8601DateFormatter', '0.7'
 pod 'ARCollectionViewMasonryLayout', '~> 2.0.0'
 pod 'SDWebImage', '~> 3.7'
 pod 'SVProgressHUD'
-pod 'RACAlertAction'
```
Comment (ashfurrow) [inline Podfile]: Moved this to be code inside the project instead. ReactiveCocoa podspec was giving me a hassle.

File: Podfile [hunk 2]
```diff
@@ -39,9 +39,7 @@ pod 'ISO8601DateFormatter', '0.7'
...
 
-pod 'HockeySDK'
```
Comment (ashfurrow) [inline Podfile]: Not ever required.

File: Podfile [hunk 3]
```diff
@@ -58,7 +56,7 @@ pod 'XNGMarkdownParser'
 
 # Swift pods
 pod 'SwiftyJSON', :git => 'https://github.com/SwiftyJSON/SwiftyJSON.git', :branch => 'xcode7'
-pod 'ReactiveCocoa', '= 3.0.0-swift2'
+pod 'ReactiveCocoa', '~> 4.0.1-alpha-2'
```
Comment (ashfurrow) [inline Podfile]: Required update.

**Comment by jorystiefel** [comment]
Looks good @ashfurrow 😎 Just be careful my github name is @jorystiefel not jory so I didn't see this right away

**Comment by ashfurrow** [comment]
Gotcha, thanks!

**Pull request merged**